manifests/overlays/kubeflow-platform/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
```

Comment:
You might need to create patch to remove this namespace:https://github.com/kubeflow/trainer/blob/7335204c3db0611672d5c82848a9a86c8dabca6b/manifests/overlays/manager/namespace.yaml

Otherwise, we can create Kubeflow Trainer resources in `kubeflow-system` namespace.

Reply:
We need to make sure that namespace is created by Kubeflow Manifests: https://github.com/kubeflow/manifests/blob/master/example/kustomization.yaml#L62
Since it contains the appropriate labels for other tools (e.g. Katib).
Thus, we should remove the namespace creation as part of Kubeflow Trainer manifests for Kubeflow Platform.

You can see example in Katib: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml#L41